test/commands/api/update.test.js

```diff
@@ -69,8 +84,8 @@ describe('invalid api:update', () => {
   test
     .stub(config, 'getConfig', () => ({ SWAGGERHUB_URL: shubUrl }))
     .nock(`${shubUrl}/apis`, api => api
-      .get('/org/api/1.0.0')
-      .reply(200)
+      .get('/org/api/1.0.0/settings/version')
```

Comment:
A lot of the test coverage for the change is provided by this, we see that if no `visibility` input is provided, the current visibility is used.